pyquil/_parser/parser.py

```diff
@@ -398,9 +395,9 @@ def move(self, left, right):
     @v_args(inline=True)
     def classical_unary(self, op, target):
         if op == "TRUE":
-            return ClassicalTrue(target)
+            return ClassicalMove(target, 1)  # TODO(andrew): should this be removed?
```

Comment:
@notmgsk Not 100% sure what to do with these...

Reply:
Leave them for now. I'll start a follow-on PR to update the parser.